src/components/NucleotideEntropyMultiChart.tsx

```diff
@@ -0,0 +1,294 @@
+import React, { useState, useMemo } from 'react';
+import { MutationProportionData } from '../data/MutationProportionDataset';
+import Loader from './Loader';
+import { useQuery } from '../helpers/query-hook';
+import { MutationProportionEntry } from '../data/MutationProportionEntry';
+import { LapisSelector } from '../data/LapisSelector';
+import { PipeDividedOptionsButtons } from '../helpers/ui';
+import { XAxis, YAxis, Tooltip, Legend, ResponsiveContainer, LineChart, Line, } from 'recharts';
+import Checkbox from '@mui/material/Checkbox';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { SequenceType } from '../data/SequenceType';
+import { Form } from 'react-bootstrap';
+import { globalDateCache, UnifiedDay } from '../helpers/date-cache';
+import { FixedDateRangeSelector } from '../data/DateRangeSelector';
+import { DateRange } from '../data/DateRange';
+import jsonRefData from '../data/refData.json';
+import { GeneOption, WeeklyMeanEntropy, TransformedTime } from './NucleotideEntropy';
+import { useExploreUrl } from '../helpers/explore-url';
+import { pprettyColors } from '../helpers/colors';
+import { getTicks } from '../helpers/ticks';
+import { formatDate } from '../widgets/VariantTimeDistributionLineChartInner';
+
+type Props = {
+    selectors: LapisSelector[];
+  };
+
+type Variant = {
+  variant: string;
+}
+
+const genes = jsonRefData.genes;
+!genes.map(o => o.name).includes('All') &&
+  genes.push({ name: 'All', startPosition: 0, endPosition: 29903, aaSeq: ''});
+
+function nonNullValues(obj: any) {
+  return Object.entries(obj).filter(([key, value]) => value !== undefined).map(([key, value]) => value).toString();
```

Comment:
Calling unused variables `_` is a common pattern, if you somehow have to include them, but you don't actually need them:
```suggestion
  return Object.entries(obj).filter(([_, value]) => value !== undefined).map(([_, value]) => value).toString();
```